roofit/roostats/CMakeLists.txt

```diff
@@ -141,3 +149,17 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RooStats
     Graf
     Gpad
 )
+
+target_include_directories(RooStats
+  PRIVATE
+  ${CMAKE_SOURCE_DIR}/io/io/res
+)
+
+
+if(yml)
+  FIND_PACKAGE(ryml)
```

Comment:
This should go to https://github.com/root-project/root/blob/master/cmake/modules/SearchInstalledSoftware.cmake#L255
Please take an example from xxhash as well as to add support of option fail-on-missing and possibly builtins in the future.